woocommerce/woocommerce-gutenberg-products-block
# Allow inserting blocks to filled mini cart

Fixes #4856 

This PR allows inserting blocks to the filled mini cart content view.
- The Title and Footer blocks are locked to the top and bottom of the filled view. 
- A new block called Mini Cart Items was created. The Products Table block now becomes an inner block of the Mini Cart Items.
- Users can add more blocks to the Mini Cart Items.
- Inner blocks of the Mini Cart Items can be moved, including the Products Table block.
- The Products Table block always takes all of the remaining vertical space to match the design.
- The Mini Cart Title font size now is the same between the editor and the front-end.

This PR also fixes the block toolbar position issue by removing `__experimentalExposeControlsToChildren` support.

### Screenshots



![image](https://user-images.githubusercontent.com/5423135/148583562-bc0354fb-49b5-4691-ac2f-d6b16e653701.png)

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Clear customization for the Mini Cart template part.
2. Edit the Mini Cart template parts.
3. See the Block Appender of the Mini Cart Items work as expected.
4. See the inner blocks of the Mini Cart Items block can be moved up and down.
5. See the Products Table block that can't be removed.
6. Save the template part and see it works correctly on the front end.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
